frontend/src/js/preview/tableUtils.ts

```diff
@@ -20,7 +15,18 @@ export function useCustomTableRenderers(queryData: GetQueryResponseDoneT) {
   );
 
   const getRenderFunction = useCallback(
-    (cellType: string): ((value: CellValue) => string) | undefined => {
+    (cellType: string): ((value: CellValue) => string) => {
+      const dateFormatter = new Intl.DateTimeFormat(navigator.language, {
```

Comment:
Good idea actually to use this new API instead of our usual `dateHelper` functions.